pkg/core/interop_system.go

```diff
@@ -282,7 +282,7 @@ func runtimeNotify(ic *interop.Context, v *vm.VM) error {
 	ne := state.NotificationEvent{
 		ScriptHash: v.GetCurrentScriptHash(),
 		Name:       name,
-		Item:       stackitem.NewArray(args),
+		Item:       stackitem.DeepCopy(stackitem.NewArray(args)).(*stackitem.Array),
```

Comment:
I think we can avoid `NewArray` here, `elem.Item()` should be enough.

Reply:
Because argument can be a struct, and we always store an array.